CMSgov/onemac
# OY2 8077 Updated List of Attachments and Sequence

Story: https://qmacbis.atlassian.net/browse/OY2-8077
Endpoint: https://d24tpyfovzs0b4.cloudfront.net
 
### Changes
- Updated all forms with new attachment lists, requirements, and label language
- Updated FileUploader to add the “At least one file” message when there are no files in the requiredUploads list.
- Vertically centered the uploader tds so the multiple-line attachment labels don’t make things look too wonky
- UserPage had a function dependency that was causing errors, also had to move function up in the hook.

### Test Plan
1. Login as stateuseractive@cms.hhs.local
2. Select each form submission link and compare the attachments list with the lists in this story’s AC
3. For “Submit 1915(b) Waiver Action” verify that the required text is changed to match the AC
4. attempt to submit a Waiver Action without including an attachment and verify you get the attachments error message  
5. attach one or more files to the attachments list and verify successful submission.
